server/views/pages/induction/checkYourAnswers/partials/_educationAndTraining.njk

```diff
@@ -50,7 +54,9 @@
     </dt>
     <dd class="govuk-summary-list__value" data-qa="additionalTraining">
       {% for item in inductionDto.previousTraining.trainingTypes %}
-        <div data-qa="additionalTraining-{{ loop.index }}">{{ item | formatAdditionalTraining }}{{ ' - ' + inductionDto.previousTraining.trainingTypeOther if item === 'OTHER' }}{{ ',' if inductionDto.previousTraining.trainingTypes.length !== loop.index }}</div>
+        <div data-qa="additionalTraining-{{ item }}">
```

Comment:
Most of the changes from here on in are essentially the same thing, in that the data-qa attribute of values being rendered on the Check Your Answers page has been changed so that it no longer uses the loop index, and instead uses the value itself.
EG. we change `data-qa=additionalTraining-1` to `data-qa=additionalTraining-HGV_LICENCE`

The reason for this is so that the cypress tests can look for the presence of a specific value using the enum rather than the rendered screen label (which might change)